aixplain/modules/cost.py

```diff
@@ -0,0 +1,51 @@
+__author__ = "lucaspavanelli"
+
+"""
+Copyright 2022 The aiXplain SDK authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Author: Duraikrishna Selvaraju, Thiago Castro Ferreira, Shreyas Sharma and Lucas Pavanelli
+Date: June 14th 2023
+Description:
+    Cost Class
+"""
+
+from typing import Dict
+
+
+class Cost:
```

Comment:
Should this be specific to FinetuneCost or FinetunePrice?

Reply:
Initially, I considered having a parent class called `Cost` with each asset implementing its estimated cost. But this class is specific to FineTune so it can be renamed to FinetuneCost.